notesapi/v1/tests/test_views.py

```diff
@@ -126,6 +139,29 @@ def test_create_note(self, mock_datetime):
 
         self.assertEqual(response.data['user'], TEST_USER)
 
+    @patch('django.conf.settings.ES_DISABLED', True)
```

Comment:
yes, and I have added third configuration mode to run our tests with: no ES. In this mode search would work slower (full-scan), less smart about analyzing phrazes and does not do highlighting. All other tasks (creation, modifying, searching by user_id, course_id, usage_id) work.